etc/config/c++.amazon.properties

```diff
@@ -3697,12 +3697,40 @@ libs.xtl.versions.053.path=/opt/compiler-explorer/libs/xtl/0.5.3/include
 libs.xtl.versions.0416.version=0.4.16
 libs.xtl.versions.0416.path=/opt/compiler-explorer/libs/xtl/0.4.16/include
 
+libs.ztdcuneicode.name=ztd.cuneicode
+libs.ztdcuneicode.url=https://github.com/soasis/cuneicode
+libs.ztdcuneicode.description=Robust C-style text encoding library<br /><br />Note that to use, you must add the "ztd.idk" library
+libs.ztdcuneicode.versions=main
+libs.ztdcuneicode.versions.main.version=main
+libs.ztdcuneicode.versions.main.path=/opt/compiler-explorer/libs/ztdcuneicode/main/include
+
+libs.ztdencodingtables.name=ztd.encoding_tables
+libs.ztdencodingtables.url=https://github.com/soasis/encoding_tables
+libs.ztdencodingtables.description=Collection of encoding tables for C and C++ programs to handle most table-driven and other styles of single and multi-byte encodings<br /><br />Note that to use, you must add the "ztd.idk" library
+libs.ztdencodingtables.versions=main
+libs.ztdencodingtables.versions.main.version=main
+libs.ztdencodingtables.versions.main.path=/opt/compiler-explorer/libs/ztdencodingtables/main/include
+
+libs.ztdidk.name=ztd.idk
+libs.ztdidk.url=https://github.com/soasis/idk
+libs.ztdidk.description=Industrial Development Kit (IDK) for core utilities (bit functions, comparison functions, and more) found across software engineering disciplines
+libs.ztdidk.versions=main
+libs.ztdidk.versions.main.version=main
+libs.ztdidk.versions.main.path=/opt/compiler-explorer/libs/ztdidk/main/include
```

Comment:
I think you need `libs.ztdidk.staticliblink=ztd.idk` - although I've never seen libs with a dot in the name, so not entirely sure... well whatever .a file this generates, the name of that minus the lib prefix

Reply:
Oooh, gotcha. I'll add that to it!